src/components/molecules/PageFilter/helpers/pagefilter.helpers.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/ban-types */
+import moment from "moment";
+
+import { stringIsNumber } from "@app/helpers/util.heplers";
+
+import { ParseFiltersProps } from "../types/pagefilter.types";
+
+interface ParseFilterParams<T> extends ParseFiltersProps<T> {
+  filters: Record<string, unknown>;
+}
+
+const isArrayIncludes = <T>(array: unknown, key: unknown) =>
```

Comment:
Will this maybe help TS to know that it only allows arrays?
```suggestion
const isArrayIncludes = <T>(array: unknown[], key: unknown) =>
```